Jenkinsfile

```diff
@@ -23,10 +23,15 @@ pipeline{
     environment {
         DEPLOYMENT_ID = credentials("messenger-mobile-sdk-deployment-id")
         DEPLOYMENT_DOMAIN = 'inindca.com'
+        HOME = """${sh(
+            returnStdout: true,
+            script: 'if [ -z "$HOME" ]; then echo "/Users/$(whoami)"; else echo "$HOME"; fi'
+        ).trim()}"""
```

Comment:
🙄

Reply:
I don't know why there's no automatic HOME env var. But, gradle touching iOS related tasks barfs without a proper HOME. I'm no Jenkins wiz, so I got past the errors by making sure a HOME is set if it doesn't already exist. I'm certainly open to suggestions on how we can make that better.

Reply:
My comment was only commiseration that you had to resort to such silliness.